app/[lang]/research/page.tsx

```diff
@@ -17,9 +18,14 @@ export const metadata = createSeoMeta({
 export default async function Research({ params }: Readonly<Language>) {
   const { lang } = await params;
   setRequestLocale(lang);
+
+  const pageService = await createRequestContainer().resolve('pageService');
+
+  const [page] = await Promise.all([pageService.getPageData('research', lang)]);
```

Comment:
```suggestion
const page = await pageService.getPageData('research', lang);
```
`Promise.all([])` is not needed here since only one promise is awaited
You can use a simple `await` instead

Reply:
thanks, fixed that